core/connectionlistmodel.cpp

```diff
@@ -39,9 +39,12 @@ void ConnectionListModel::addAddress(const QString &address)
 
 void ConnectionListModel::removeAllAddresses()
 {
-	beginRemoveRows(QModelIndex(), 0, rowCount());
+	if (rowCount() == 0)
```

Comment:
With `beginResetModel()`/`endResetModel()` the check is redundant.

Reply:
it's redundant. Is it wrong?